README.md

```diff
@@ -16,20 +16,31 @@ Ami app gives you flexibility to test multiple features such as :
 
 
 # Getting Started
+
 Ami app already includes all the dependencies required. To install the libraries using :
 ```
-npm install
+yarn
 ```
-**iOS only**
+
+## iOS setup
 
 iOS needs an additional step to install libraries using CocoaPods
 ```
 cd ios
 pod install
 cd ..
 ```
+
+Next, [follow these steps](https://github.com/customerio/RemoteHabits-iOS/blob/main/docs/dev-notes/DEVELOPMENT.md#setup-code-signing-internal-team-member-only) to download the provisioning profiles and certificates for development to your machine. 
```

Comment:
Now that we have separate provisioning profile, bundle id & certificates for Ami app, I think we should create a similar document for Ami app to instruct how to setup code signing for it. What do you say ?

Reply:
Assuming I have time this week, I was going to create a generic code signing document that RH and Ami app can both reference. 

This is a solution that works good enough. The documentation at `https://github.com/customerio/RemoteHabits-iOS/blob/main/docs/dev-notes/DEVELOPMENT.md#setup-code-signing-internal-team-member-only` still works for Ami app. The commands that the document tells you to run are the same in the Ami project so the instructions will still work here.